ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1800,6 +1800,68 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: baremetalds-ipi-ovn-ipv6-fips-f14
+  cron: 52 13 2,17 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      DEVSCRIPTS_CONFIG: |
+        FIPS_MODE=true
+        IP_STACK=v6
+        MIRROR_IMAGES=true
+        INSTALLER_PROXY=true
+        NETWORK_TYPE=OVNKubernetes
+      E2E_RUN_TAGS: '@amd64 and @baremetal-ipi and @network-ovnkubernetes'
+      TAG_VERSION: '@4.12'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: baremetalds-ipi-ovn-ipv6-fips-f28-destructive
+  cron: 49 12 22 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      DEVSCRIPTS_CONFIG: |
+        FIPS_MODE=true
+        IP_STACK=v6
+        MIRROR_IMAGES=true
+        INSTALLER_PROXY=true
+        NETWORK_TYPE=OVNKubernetes
+      E2E_RUN_TAGS: '@amd64 and @baremetal-ipi and @network-ovnkubernetes'
+      TAG_VERSION: '@4.12'
+      TEST_ADDITIONAL: Network_Observability
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: baremetalds-ipi-ovn-dualstack-ipv4v6-fips-f14
```

Comment:
I guess we can remove `ipv4v6` here.

Reply:
If it is stressing the primary IP stack, then ignore the comment.

Reply:
since 4.12 only support ipv4v6, and we have v4v6 and v6v4 supported from 4.13.  just want to make this consistent.

Reply:
For vSphere we used `dualstack` to mean `v4v6` and dualstack-primaryv6` to mean `v6v4`.

We should work on the naming convention.  For clusterbot we have

```
"dualstack", "dualstack-primaryv6", 
```
https://github.com/openshift/ci-chat-bot/blob/master/pkg/manager/prow.go#L59

Reply:
This matches MCO 
```go
const (
	IPFamiliesIPv4                 IPFamiliesType = "IPv4"
	IPFamiliesIPv6                 IPFamiliesType = "IPv6"
	IPFamiliesDualStack            IPFamiliesType = "DualStack"
	IPFamiliesDualStackIPv6Primary IPFamiliesType = "DualStackIPv6Primary"
)
```

Reply:
yeah, agree, my initial comments was just from the vsphere naming convention.

Reply:
And even more, per `workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4`, if the install workflow can work for dualstack cases, I think we even should introduce a new install workflow, something like `cucushift-installer-rehearse-baremetalds-ipi-ovn` to avoid confustion.

Reply:
ok, updated the job title to make name consistent
ipv4v6 --> dualstack  
ipv6v4 --> dualstack-primaryv6

Reply:
> And even more, per `workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4`, if the install workflow can work for dualstack cases, I think we even should introduce a new install workflow, something like `cucushift-installer-rehearse-baremetalds-ipi-ovn` to avoid confustion.

`cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4` this workflow can work for all ipv4, ipv6, DualStack, DualStackIPv6Primary according to the parameter `IP_STACK`.   

I think we can update the workflow name `cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4` to `cucushift-installer-rehearse-baremetalds-ipi-ovn` if you feel that confusion.

Reply:
Yeah, I prefer to remove the `ipv4` from the install workflow to avoid confusion

Reply:
@jianlinliu Done